hbase-it/src/test/java/org/apache/hadoop/hbase/chaos/monkies/PolicyBasedChaosMonkey.java

```diff
@@ -151,7 +157,9 @@ public boolean isStopped() {
 
   @Override
   public void waitForStop() throws InterruptedException {
-    monkeyThreadPool.awaitTermination(1, TimeUnit.MINUTES);
+    if (!monkeyThreadPool.awaitTermination(1, TimeUnit.MINUTES)) {
+      LOG.warn("Some pool threads failed to terminate, {}", monkeyThreadPool);
```

Comment:
Strange. `monkeyThreadPool` is not a daemon pool, so if this termination fails for some reason, the process can hang. Let me add a forced shutdown.